### PR TITLE
allure 2.0.1 (new formula)

### PR DESCRIPTION
**Pull request by eroshenkoam**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/allure.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Allure < Formula
+  desc "Flexible lightweight test report tool"
+  homepage "https://github.com/allure-framework"
+  url "https://dl.bintray.com/qameta/generic/io/qameta/allure/allure/2.0.1/allure-2.0.1.tgz"
+  sha256 "315f2a9e963ce6293075149b15ed11423ac3709adb03658d1b16ed59bafcaa6e"
+
```
Comment (fxcoudert) [inline Formula/allure.rb]: Will need `bottle :unneeded` line

File: Formula/allure.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Allure < Formula
+  desc "Flexible lightweight test report tool"
+  homepage "https://github.com/allure-framework"
+  url "https://dl.bintray.com/qameta/generic/io/qameta/allure/allure/2.0.1/allure-2.0.1.tgz"
```
Comment (fxcoudert) [inline Formula/allure.rb]: Could we use the github release tarball?

File: Formula/allure.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class Allure < Formula
+  desc "Flexible lightweight test report tool"
+  homepage "https://github.com/allure-framework"
```
Comment (fxcoudert) [inline Formula/allure.rb]: Maybe `https://github.com/allure-framework/allure2` makes more sense as a homepage?

File: Formula/allure.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    # Remove all windows files
+    rm_f Dir["bin/*.bat"]
+
+    prefix.install_metafiles
+    libexec.install Dir["*"]
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system "#{bin}/allure", "--help"
```
Comment (fxcoudert) [inline Formula/allure.rb]: We want a test that does a little bit more than output the help :)

**Review by fxcoudert — CHANGES_REQUESTED** [review]
Please squash to get a single-commit PR

File: Formula/allure.rb (hunk 3 above)
Comment (eroshenkoam) [inline Formula/allure.rb]: done

File: Formula/allure.rb (hunk 1 above)
Comment (eroshenkoam) [inline Formula/allure.rb]: done

File: Formula/allure.rb (hunk 4 above)
Comment (eroshenkoam) [inline Formula/allure.rb]: done

File: Formula/allure.rb (hunk 2 above)
Comment (eroshenkoam) [inline Formula/allure.rb]: We are prefer using bintray. It's bad place for binaries?

**Review by fxcoudert — APPROVED** [review]

**Pull request merged**